react-locate-contractor/src/Services/BusinessService.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import axios from 'axios';
+import { business } from "../Pages/BusinessCard";
+import { Business } from "@mui/icons-material";
+
+export const getBusiness = async (searchString: string) => {
+    return await (await axios.get(`http://localhost:5149/Business/email/${searchString}`));
```

Comment:
Typically you would want to abstract out the base url 'http://localhost:5149' to your environment or config.